src/Hill.jl

```diff
@@ -42,19 +42,34 @@ end
 function optimize_helper(f, g!, low::Vector, high::Vector, maxstep::Int)
     initial_x = low + (high - low) / 2.0
 
-    options = Optim.Options(outer_iterations = 2, show_trace = true, iterations = maxstep)
-    results = optimize(f, g!, low, high, initial_x, Fminbox(LBFGS()), options)
-
-    println(results)
-
-    return Optim.minimum(results), Optim.minimizer(results)
+    function my_func(x, grad)
+        grad[:] = g!
+        f(x)
+    end
+    P = MinimizationProblem(f, low, high)
+    opt = Opt(:LD_MMA, 20)
+    opt.min_objective = my_func
+    opt.xtol_rel = 1e-4
+    # local_method = NLoptLocalMethod(NLopt.LD_MMA)
+    multistart_method = TikTak(100)
+    p = multistart_minimization(multistart_method, opt, P)
+
+    return p.location, p.value
```

Comment:
Is this close to that interface?
It errors that we cannot pass `opt` to the `mulitstart_minimization`. It should be the type `!Matched::MultistartOptimization.NLoptLocalMethod`. 
![image](https://user-images.githubusercontent.com/34812750/105569639-193cec80-5cf8-11eb-859f-48c67410910b.png)